parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -87,9 +87,12 @@ fi
 {{end}}
 
 if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
-  if apt list --installed | grep 'ntp'; then
-    time_metric "EnsureNTP" ensureNTP
-  fi
+  if [[ ! $(apt list --installed | grep 'chrony') ]]; then
+    apt_get_install 30 1 600 chrony || exit 9
```

Comment:
I don't think we wanna apt-get install this again (it's being installed in the `installDeps` flow), but instead we wanna do the `configureChrony` and `ensureChrony` steps in this if block

Reply:
This is to support "bring your own Ubuntu 18.04 flavor image" scenario, and we may need it for airgap cloud until newer VHDs are built and become available.

Reply:
Can you describe that rare scenario? I'm pretty sure installDeps will be called in a BYO 18.04 scenario. If you customize your image from the AKS Engine VHD, it will have already been called during VHD CI; if you customize your image some other way, the `/opt/azure/vhd-install.complete` sentinel file won't exist, and you will invoke `installDeps` at runtime (during CSE).

Reply:
Thanks for the explanation, didn't realize installDeps would be called for BYO situation. I will remove these lines.